config/credentials.yml.enc

```diff
@@ -1 +1 @@
-cPAVs9uvfnXY5UKBEhahofQDQiJYIc52Oyme3pm46LJdH0fPi76q8bVKX2D5giPmFxSYFOJH/KcaQj3YVbcEM9crss7MMH6aZijhmzFBbB4TJxqXzYyZxPvLj4BH0H726+u576jSFI1epDJiq4HElxrNjlIAvD3/ZMwSUqYrmuP9IEZJlUnLzIdbdUO0Ha0Zv3Z4Cgj9LXftT0WbR2MQgMrjeasMjTXDVFfuk55daUEdXV4rTR9q57bcC3TjH8jUDTr+Y5ch0mjRvVDUMb3QADU0XTAQE5Knk6TpRiuYxKkQT/bVGrTvT0sDtLptj1ibe5RjI9qu+AQB69B1F1PKcQLYiU8XA7q9gtgUrq6p3xobEwqXd1nixnIjnUC5Es+usmegMOWpo2XLAhp89F+BXT9maQnxAaYo3782--bKh9IhvsU8iZiXoh--PinXhZUr4C/5j5wwidXSmg==
\ No newline at end of file
+zglugiAAHXpSJQLoQcB97MeWy2d6pOW3+BxortH9rcLsCAGsoOCh1xSAVa+8dbbxZh6k1g0IY9aq8xXv7edhhizfLSA8hXOwjOtnint0dZAHIHtVKrqKADsCz6bmtMAjpciQsit5FEvxmqDTePW+nf2rRR/i+dGwOl4dFVhv41hq9krxgml1RuO57Gg88xjlvaf6Hnl2HRRtQwuLGE0rShI9M++gH7UAtZkhgCjivOzU+BIeArTB4L4EV3LiXOe3U87ITMxwi+DSuSatqxQQa1nx7BuCzY0m8NQDX8RKMSjAWN7vtpumUmfzSySdKpo580c24E9nVi6FhXUgKC+B/0D9Y4HHcZXD18hw4s87hNl6hbVErMTSATjGc3RRA2FSmqK8Vm8ElaHFapM1GGnZl7Y6xEc88IVLJ1pP--kk89gdumRN8OVWJ0--q1zVq95IGfgiOnWyEajhuQ==
```

Comment:
please add this file to gitignore
and create credentials.yml.enc.example instead